### PR TITLE
updates to 'About' page

### PR DESCRIPTION
**Pull request by cgmeyer**
facelift to gen3.org based on 2021 overview: https://docs.google.com/document/d/1NinAx8DgKox_7Vup1pXXfxVAXtwtAVrGnvtWjbbLMFk/edit#heading=h.k2ftb1zif9d

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -73,40 +69,42 @@ g3Commons:
     link: https://www.bloodpac.org/
   c3:
     fig: figs/commons/pdc.png
-    link: /
-  c4:
-    fig: figs/commons/braincommons.png
```
Comment (mfshao) [inline content/_index.md:78]: can also delete this file from `/figs`

**Review by wangfan860 — APPROVED** [review]
LGTM!👍

File: content/_index.md [hunk 2]
```diff
@@ -73,40 +69,42 @@ g3Commons:
...
-    link: https://www.braincommons.org/
+    link: https://proteomic.datacommons.cancer.gov/pdc/
 g3Intro:
   title: Introduction to Gen3
-  detail: Gen3 is an open source software with Apache 2.0 or similar licenses, colocating compute and storage in a data commons. It is agnostic to the data type and the storage location, requiring  minimally, a data model, data, a secure landing page for the portal, and research goals.
+  detail: The Gen3 Data Platform is an open-source (with an Apache License) data platform for developing and operating cloud-based data commons, workspaces, and an ecosystem of interconnected, cloud-based data resources, applications and notebooks. A functioning data ecosystem empowers researchers in the same field to share and combine datasets for reanalysis and reuse as new tools and hypotheses are developed.  The Gen3 Data Platform consists of Gen3 Framework Services, Gen3 Data Commons and Gen3 Workspaces.
   fig: figs/productkv.svg
```
Comment (xritter1) [inline content/_index.md:76]: Saw that you used the Oxford Comma above so: 
"applications, and notebooks".
"Gen3 Data Commons, and Gen3 Workspaces."

I might also suggest to change the first sentence into:
"...an open-source data platform with Apache License..."

And, really minor: two spaces before the beginning of the last sentence

File: content/_index.md [hunk 3]
```diff
@@ -73,40 +69,42 @@ g3Commons:
...
+g3DataCommons:
+  title: Gen3 Data Commons
+  para1: Gen3 Data Commons are cyberinfrastructure that co-locates data analysis, exploration and visualization tools with data management services for import and export of structured information like clinical, phenotypic, or biospecimen data, and data objects, like genomics data files or medical images. Typically, a data model is utilized to standardize the structure of metadata input and output, although a commons may also support multiple data models by allowing submission of structured data packaged along with a custom data model, for example, Avro-based data-serialization formats like the Portable Format for Biomedical data or PFB.
+  para2: When using Gen3 to build a data commons, the platform auto-generates a data submission and exploration portal that enables the use of its features within a standard web-browser. Fully-integrated analysis workspaces can be also be included, although data can be exported to virtually any cloud-based analysis platform that allows programmatic access via Gen3 open APIs.
```
Comment (xritter1) [inline content/_index.md:86]: "can be also be included, " -- two "be"s

**Review by svburke — APPROVED** [review]

File: content/_index.md [hunk 4]
```diff
@@ -73,40 +69,42 @@ g3Commons:
...
-  para2Fig: img/figs/data-access-diagram.png
-  para3: Gen3 also has an Attribute Based Access Control (ABAC) engine. It can be used to define data access controls and permissions on a more granular level, and can determine if a user is able to access a specific piece of data. Data in our graph database is modeled as a hierarchy that starts with Programs and Projects. A Program is the root of the tree, and represents an overall group for the data. A Program will have Projects underneath it that consist of any subgroups. Projects can then have many subgroups, and so on. Permissions are generally given on the Program level, but with the ABAC engine, Gen3 can provide more specific authorization that applies to nodes further down the tree.
-  para3Fig: img/figs/data-model-diagram.png
```
Comment (xritter1) [inline content/_index.md:93]: delete from folder /figs too if not used?: img/figs/data-model-diagram  and img/figs/data-access-diagram.png

File: content/_index.md [hunk 5]
```diff
@@ -73,40 +69,42 @@ g3Commons:
...
+  para2: When using Gen3 to build a data commons, the platform auto-generates a data submission and exploration portal that enables the use of its features within a standard web-browser. Fully-integrated analysis workspaces can be also be included, although data can be exported to virtually any cloud-based analysis platform that allows programmatic access via Gen3 open APIs.
+  para3: Gen3 Data Commons are a kind of data resource that is interoperable with other nodes in a data ecosystem by utilizing the Gen3 Framework Services for authentication, authorization, data object indexing and metadata services.
+g3Workspaces:
+  title: Gen3 Workspaces
+  para1: A Gen3 workspace can access data from one or more data resources, including Gen3 Data Commons, and also uses the Gen3 Framework Services for user authentication and authorization and data object indexing and metadata services. Gen3 Workspaces support Jupyter notebooks, RStudio notebooks, and other applications that can access data through Gen3 open APIs.
```
Comment (xritter1) [inline content/_index.md:90]: I'd suggest to remove some "ands":
"A Gen3 workspace can access data from one or more data resources, including Gen3 Data Commons, and also uses the Gen3 Framework Services for user authentication, authorization, data object indexing, and metadata services."

File: content/_index.md [hunk 6]
```diff
@@ -73,40 +69,42 @@ g3Commons:
...
+  para2: When using Gen3 to build a data commons, the platform auto-generates a data submission and exploration portal that enables the use of its features within a standard web-browser. Fully-integrated analysis workspaces can be also be included, although data can be exported to virtually any cloud-based analysis platform that allows programmatic access via Gen3 open APIs.
+  para3: Gen3 Data Commons are a kind of data resource that is interoperable with other nodes in a data ecosystem by utilizing the Gen3 Framework Services for authentication, authorization, data object indexing and metadata services.
```
Comment (xritter1) [inline content/_index.md:87]: I'd suggest:
"Gen3 Data Commons are a flexible data resource.." or so? Or "extendable"? "powerful"? :-)

File: content/get-started.md [hunk 7]
```diff
@@ -6,30 +6,30 @@ date: 2018-09-04T22:16:21-05:00
...
+    detail: Have data to contribute to a Gen3 system? Check out our documentation on how to submit metadata or data objects for ingestion.
     button:
       caption: Data Contribution
       link: /resources/user/submit-data
   s3:
     name: Access Metadata & Data
     fig: /figs/features/search.svg
-    detail: Gen3 includes a data portal as a default application over a commons. The portal is an interactive website that allows users to explore, submit, and download data. The data portal utilizes the public APIs offered by the data commons as a demonstration to the power of Gen3. Or look into how to build your own submission apps.
+    detail: Gen3 includes a portal service for providing a web-based graphical user interface for interacting with a Gen3 data commons within a standard web-browser. The portal is an interactive website that allows users to explore, submit, download and analyze data. The data portal utilizes the public APIs offered by the data commons as a demonstration of the power of Gen3.
```
Comment (xritter1) [inline content/get-started.md:25]: "download, and analyze data"

**Review by xritter1 — APPROVED** [review]
Great stuff!

File: content/_index.md (hunk 4 above)
Comment (cgmeyer) [reply content/_index.md:93]: yeah, we may re-use the images and text elsewhere though. shouldn't hurt anything to keep the PNGs in the folder.

**Pull request merged**